main.css

```diff
@@ -193,3 +193,7 @@ section {
     padding: 1em;
     text-decoration: none;
 }
+
+#calendar {
+    text-align: center;
+}
```

Comment:
Smazat prosim (kontext viz muj dalsi komentar v `main.css`)

Reply:
Ha, tak neni mozne komentovat neco, co GitHub nevidi jako zmenu, takze to napisu zde.

Styly pro kalendar presun sem [main.css radek 107](https://github.com/klubFITpp/klubfitpp/pull/32/files#diff-928f8ef12735c60b1dc8b4e5f874eef2c90c9ea3a781525147a86c137d4238c4R107), at je to v tom CSSku serazeny jak to je videt na webu, bude v tom mensi bordel.

Zaroven bych kalendar schoval pro telefony, na kterych se afaik nezvladne vykreslit spravne a styly pro iframe presunul do CSS, at jsou vsechny styly na jednom miste. Styly staci asi zkopirovat jen z `.about`

Celkove styly bych udelal takhle:

```css
/* calendar */
.calendar {
    display: flex;
    flex-direction: column;
    align-items: center;
    justify-content: center;
}

.calendar__iframe {
    width: 100%;
    max-width: 1024px;
    height: 600px;
    border: 0;
}

@media screen and (max-width: 1280px) {
    .calendar {
        display: none;
    }
}